data/xml/1993.iwpt.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1993.iwpt">
+  <volume id="1">
+    <meta>
+      <booktitle>Third International Workshop on Parsing Technologies</booktitle>
+      <publisher>ACL</publisher>
+      <address>Tilburg, Netherlands and Durbuy, Belgium</address>
+      <month>August 10-13</month>
```

Comment:
Why is Durbuy here? [The official page](https://www.sigparse.org/meetings.html) lists Tilburg only.

Reply:
second part of conference was in Durbuy